python/ray/tune/trial_runner.py

```diff
@@ -1081,18 +1096,11 @@ def _process_trial_restore(self, trial):
             trial (Trial): Trial being restored.
         """
         logger.debug("Trial %s: Processing trial restore.", trial)
-        try:
-            self.trial_executor.fetch_result(trial)
-            trial.on_restore()
-            logger.debug("Trial %s: Restore processed successfully", trial)
-            self.trial_executor.set_status(trial, Trial.RUNNING)
-            self.trial_executor.continue_training(trial)
-            self._live_trials.add(trial)
-        except Exception:
-            logger.exception("Trial %s: Error processing restore.", trial)
-            if self._fail_fast == TrialRunner.RAISE:
-                raise
-            self._process_trial_failure(trial, traceback.format_exc())
+        trial.on_restore()
+        logger.debug("Trial %s: Restore processed successfully", trial)
+        self.trial_executor.set_status(trial, Trial.RUNNING)
+        self.trial_executor.continue_training(trial)
+        self._live_trials.add(trial)
```

Comment:
Why did we remove the try-except block here? Is everything caught before?

Reply:
Hmmm, let me know if I misunderstand but if you look at `_wait_and_handle_event`, there is actually a big try, catch blob.